src/babel.js

```diff
@@ -15,8 +15,7 @@ module.exports = babelJest.default.createTransformer({
 		[
 			'@babel/plugin-transform-react-jsx',
 			{
-				pragma: 'h',
-				pragmaFrag: 'Fragment',
+				runtime: 'automatic'
```

Comment:
```suggestion
				runtime: 'automatic',
				importSource: 'preact'
```

This would not work without setting the import source.